Assets/Scripts/SS3D/Systems/Inventory/Items/ItemActor.cs

```diff
@@ -91,20 +82,24 @@ public Sprite InventorySprite
         /// </summary>
         public Container Container
         {
-            get => _container;
+            get => item.Container;
             set => SetContainer(value, false, false);
         }
 
-        public void Awake()
+        public new void Awake()
         {
-            _sprite = null;
+            item.Name = _startingName;
```

Comment:
use a constructor instead of setters, goal is to keep those fields private and not easily modifiable (with some logic at least, not just freely)